src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
 // write your code here
+
+const logotype = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+function createPromises() {
+  const resolver = (resolve, reject) => {
+    logotype.addEventListener('click', () => {
+      resolve('Succes');
+    });
+
+    setTimeout(() => {
+      reject('Error!!!');
+    }, 3000);
+  };
+
+  return new Promise(resolver);
+};
+
+const promise1 = createPromises();
+const promise2 = createPromises();
```

Comment:
now you have 2 identical  promises
just create 2 separate promises (1 for click, 2 for timeout)